src/tests/test_complete_dashboard.py

```
@@ -0,0 +1,8 @@
+import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

(F401)